/vel_estimation:
  ros__parameters:
    base_link_frame_id: "base_link"
    odom_frame_id: "odom"
    contact_joint_names:
      - FL_FOOT
      - FR_FOOT
    contact_timeout: 0.03
    joint_state_timeout: 0.03
    zero_velocity_timeout: 0.2

    vel_std_0: 0.1
    att_std_0: 0.1
    acc_bias_std_0: 0.0001
    gyro_bias_std_0: 0.00001
    
    acc_noise_density: 0.001
    gyro_noise_density: 0.001
    acc_bias_random_walk: 0.0001
    gyro_bias_random_walk: 0.0001

    zero_vel_noise_cov: 0.2
    contact_vel_covariance: 0.1